languages/sql/index.json

```
@@ -5,6 +5,37 @@
             "url": "https://sqlbolt.com/",
             "type": "website",
             "level": "everyone"
+        },
+        {
+            "title": "sql-practice",
+            "url": "https://www.sql-practice.com/",
+            "type": "website",
+            "level": "everyone"
+        },
+        {
+            "title": "SQL in one page",
+            "url": "https://www.cheat-sheets.org/sites/sql.su/",
+            "type": "website",
```

We have a cheatsheet category :)
```suggestion
            "type": "cheatsheet",
```